Python/tier2.c

```diff
@@ -36,6 +36,83 @@ PyTypeObject PySmallInt_Type = {
 
 static inline int IS_SCOPE_EXIT_OPCODE(int opcode);
 
+////////// TYPE NODES FUNCTIONS
+
+/**
+ * @brief Maps _Py_NegativeTypeMaskBit to typeobject
+ * @param bitidx _Py_NegativeTypeMaskBit
+ * @return Corresponding typeobject
+*/
+PyTypeObject *bit_to_typeobject(int bitidx)
```

Comment:
PEP 7 nits, same for all the new functions below
```suggestion
PyTypeObject *
bit_to_typeobject(int bitidx)
```

Reply:
Okie done